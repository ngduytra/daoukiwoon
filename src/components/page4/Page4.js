import React from 'react';
import './page4.styles.scss'
import Computer from '../../assets/img/Computer.png'

const list = [
    'Webmail',
    'E-Apporval',
    'E-Document',
    'E-Check in.out',
    'Co-work/Project'
]

const Page2 = () => {
    return (
        <div id='index-4'>
            <div className="content">
                <img src={Computer} />
                <div className='right'>
                    <div className='title'>HR & ADMIN</div>
                    <div className='tag-container'>
                    {
                        list.map((value, idx) => (
                            <div className="tags" key={idx}>
                                <div className='subtitle'>{value}</div>
                                <div className='button'>G-Book</div>
                            </div>
                        ))
                    }
                    </div>
                </div>
            </div>
        </div>
    )
}

export default Page2;