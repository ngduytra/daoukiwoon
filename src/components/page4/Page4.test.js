import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import React from 'react';
import Page4 from './Page4';

configure({adapter: new Adapter()});

it('expect to render Page4 component', ()=>{
    expect(shallow(<Page4/>)).toMatchSnapshot();
})