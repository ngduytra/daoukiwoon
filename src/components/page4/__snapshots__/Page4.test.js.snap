// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Page4 component 1`] = `ShallowWrapper {}`;
